src/main/resources/application.yaml

```
@@ -5,17 +5,60 @@ spring:
+  match:
+    url: ${MATCH_SERVICE_URL:http://match-service:20002}
+  payment:
+    url: ${PAYMENT_SERVICE_URL:http://payment-service:20006}
+
   security:
     oauth2:
       resourceserver:
         jwt:
           issuer-uri: http://localhost:18080/realms/ticketing
 
+spring:
```

YAMLlint (1.38.0)

[error] duplication of key "spring" in mapping

(key-duplicates)